### DISCUSSION TITLE
How to make custom icons be seen on bedrock??

### DISCUSSION THREAD

**Discussion by ShadowJaypeeBeast** [Kas-tle/java2bedrock.sh#8592] [Converter Help]
I need help on how to let bedrock players see icon in-chat, scoreboards. I am also using oraxen

**Comment by typicalsmc** [comment]
https://wiki.bedrock.dev/ 
check the concepts > emojis section

**Comment by sajjadhasansany** [comment]
I know some about costom ui and icons. I test it in my server. But some glitch . Try to fix . Hope some day I fix this
